static/app/components/devtoolbar/components/releases/releasesPanel.tsx

```diff
@@ -41,8 +42,9 @@ import {panelInsetContentCss, panelSectionCss} from '../../styles/panel';
 import {smallCss} from '../../styles/typography';
 import PanelLayout from '../panelLayout';
 
-const estimateSize = 81;
-const placeholderHeight = `${estimateSize - 8}px`; // The real height of the items, minus the padding-block value
+const summaryPlaceholderHeight = '65px';
```

Comment:
How do you usually calculate these?

Reply:
manually........ i just eyeball and adjust the pixels until it's perfect